test/e2e/argo-workflows/templates/cpu-stress.yaml

```diff
@@ -35,10 +35,10 @@ spec:
                   resources:
                     requests:
                       memory: "64Mi"
-                      cpu: "400m"
+                      cpu: "1"
                     limits:
                       memory: "64Mi"
-                      cpu: "400m"
+                      cpu: "1"
```

Comment:
Just out of curiosity, why did we had to increase the amount of CPU that the “CPU burner” workload will eat ?
I would have thought that everything would be more predictable if other components hadn’t to fight too much against this pod to get CPU.
It is because the CFS quotas are disabled on that setup ?

Reply:
VM CPU have been increased to 4 (from 2) - to be faster and more reliable. CPU increased to have simpler values in the tests